test/e2e/config/cre.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+managementClusterName: cre-e2e
+
+images:
+  - name: ko.local/capi-runtime-extensions:${E2E_IMAGE_TAG}
+    loadBehavior: mustLoad
+
+providers:
+- name: cluster-api
+  type: CoreProvider
+  versions:
+  - name: v1.6.1
```

Comment:
We could, but in future I would prefer to use the latest patch versions which will be resolvable in the next CAPI minor release by using `{go://sigs.k8s.io/cluster-api@v0.6}` (see https://github.com/kubernetes-sigs/cluster-api/blob/main/test/e2e/config/docker.yaml#L65 for an example). This will enable our e2es to always run against latest patch releases from CAPI which is important for compatibility tests imo.

So are you ok with keeping these values like this for now and updating once the CAPI 1.7 release is out?